packages/concerto-core/lib/introspect/modelfile.js

```diff
@@ -393,13 +393,7 @@ class ModelFile {
                 }
             }
             else {
-                // check whether type is defined in another file
-                const fqn = this.resolveImport(type);
-                const modelFile = this.getModelManager().getModelFile(ModelUtil.getNamespace(fqn));
-                if (!modelFile) {
-                    return null;
-                }
-                return modelFile.getLocalType(fqn).getFullyQualifiedName();
```

Comment:
All of this code is seemingly redundant - we use the FQN that we already have to locate the model file of the type, then we get the type, and then we get the FQN of the type (which we already know!) and return that.

Reply:
Maybe worth adding the motivation for this actually - yes, it will have a small performance benefit, but it actually enables you to get fully qualified names out of a `ModelFile` instance without having to have the imported models present in your `ModelManager` now that we have versioned imports and are looking at dropping wildcard imports.